docs/versioned_docs/version-zenith/guides/snippets/create-app-ci-module/base/python/main.py

```diff
@@ -1,12 +1,22 @@
+import random
 import dagger
-from dagger import dag, function
+from dagger import dag, object_type, field, function
 
-# build base image
-def build_base_image() -> dagger.Node:
-    return (
-        dag.node()
-        .with_version("21")
-        .with_npm()
-        .with_source(dag.current_module().source())
-        .install([])
-    )
+@object_type
+class MyModule:
+    source: dagger.Directory = field()
+
+    @classmethod
+    def create(cls, source: dagger.Directory):
+        return cls(source=source)
+
+    # build base image
+    def build_base_image(self) -> dagger.Container:
+        return (
```

Comment:
[Docstrings](https://peps.python.org/pep-0257/#one-line-docstrings) in Python are in the function body, with tripple quotes:
```suggestion
    def build_base_image(self) -> dagger.Container:
        """Build base image"""
        return (
```